README.md

```diff
@@ -45,6 +45,27 @@ It aims to create a customizable, extendable and lightweight environment without
 - [wf-config](https://github.com/WayfireWM/wf-config)
 - [wlroots](https://github.com/swaywm/wlroots)
 
+### Login Provider (optional)
+
+- [systemd](https://systemd.io/) **or**
+- [elogind](https://github.com/elogind/elogind)
+
+### XWayland Support (optional)
```

Comment:
Some of these are hard to get except if you don't compile from source. For example, I do not have a package `xcb-errors`, but neither is this really required. Can you list only required packages here? (for login providers, I'd say it is okay).